pkg/subctl/cmd/deploybroker.go

```diff
@@ -83,7 +83,7 @@ var deployBroker = &cobra.Command{
 		status := cli.NewStatus()
 		status.Start("Deploying broker")
 		err = broker.Ensure(config)
-		status.End(err == nil)
+		status.End(0)
```

Comment:
This is not equivalent, may be you can have an auxiliary function that takes err and return status ?

Reply:
Done